circle.yml

```diff
@@ -8,3 +8,6 @@ test:
   override:
     - bundle exec rake
     - bundle exec appraisal rake
+machine:
+  ruby:
+    version: 2.2.3
```

Comment:
is this necessary? I would have assumed CircleCI would auto-detect the ruby version based on `.ruby-version`.